### PR TITLE
Migrate recipe for nit_song070 from Muskit

### PR DESCRIPTION
**Pull request by wwwbxy123**
The recipe from Muskit: https://github.com/SJTMusicTeam/Muskits/tree/main/egs/nit_song070/svs1

Similar to opencpop recipe. The difference is making segments in wav, label text and and score.

Successfully started training step.

### CODE REVIEW

**Comment by ftshijt** [comment]
Thanks! @A-Quarter-Mile  could you help with @wwwbxy123 's recipe formulation?

Can you also at the same time, fix the issue in CI? https://github.com/espnet/espnet/actions/runs/5352866393/jobs/9708205475?pr=5251#step:7:465

**Comment by wwwbxy123** [comment]
> Thanks! @A-Quarter-Mile could you help with @wwwbxy123 's recipe formulation?
> 
> Can you also at the same time, fix the issue in CI? https://github.com/espnet/espnet/actions/runs/5352866393/jobs/9708205475?pr=5251#step:7:465

Done.

File: egs2/nit_song070/svs1/run.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+score_feats_extract=frame_score_feats   # frame_score_feats | syllable_score_feats
+
+opts=
+if [ "${fs}" -eq 48000 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format wav "
+fi
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/run.sh:23]: We can directly use `opts="--audio_format wav "` here.

File: egs2/TEMPLATE/asr1/pyscripts/utils/format_score_scp.py [hunk 2]
```diff
@@ -40,7 +40,8 @@ def main():
                 if len(line) == 0:
                     continue
                 utt_id, _, _, _ = line.strip().split(" ")
-                fscp.write(f"{utt_id} {dic[utt_id]}\n")
+                if utt_id in dic.keys():
+                    fscp.write(f"{utt_id} {dic[utt_id]}\n")
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/asr1/pyscripts/utils/format_score_scp.py:44]: Will this change hurt the other recipes? We'd better have a double check.

File: .gitignore
Comment (A-Quarter-Mile) [inline .gitignore:1]: Is this file necessary?

File: egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp:1]: Please remove `.tmp` in file name.

File: egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+best_model_criterion:
+-   - valid
+    - loss
+    - min
+-   - train
+    - loss
+    - min
+
+gpu_id: 0
+vocoder_checkpoint: /home/exx/jiatong/projects/svs/ParallelWaveGAN/egs/opencpop/voc1/exp/tr_no_dev_opencpop_hifigan.v1/checkpoint-250000steps.pkl
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp:85]: We don't need them here.

File: egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+# num_iters_per_epoch: 200    # number of iterations per epoch
+max_epoch: 500              # number of epochs
+grad_clip: 1.0              # gradient clipping norm
+grad_noise: false           # whether to use gradient noise injection
+accum_grad: 1               # gradient accumulation
+
+# batch_bins: 1000000       # batch bins (for feats_type=fbank)
+# batch_bins: 10000000         # batch bins (for feats_type=raw, *= n_shift / n_mels)
+# batch_type: numel           # how to make batch
+
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp:57]: Please remove these lines.

File: egs2/nit_song070/svs1/conf/tuning/train_xiaoice_noDP.yaml
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/conf/tuning/train_xiaoice_noDP.yaml:1]: We don't support Xiaoice_noDP in ESPnet-Muskit. Just delete this file.

File: egs2/nit_song070/svs1/run.sh [hunk 5]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# spectrogram-related arguments
+fs=24000
+fmax=7600
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/run.sh:10]:
```suggestion
fmin=80
fmax=7600
```

File: egs2/nit_song070/svs1/run.sh [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+cleaner=none
+
+./svs.sh \
+    --lang jp \
+    --stage 1 \
+    --local_data_opts "--stage 0 $(pwd)" \
+    --feats_type raw \
+    --pitch_extract None \
+    --fs "${fs}" \
+    --fmax "${fmax}" \
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/run.sh:44]:
```suggestion
    --fmin "${fmin}" \
    --fmax "${fmax}" \
```

**Comment by A-Quarter-Mile** [comment]
Thanks for your contributions! The above are minor changes.

File: .gitignore
Comment (ftshijt) [reply .gitignore:1]: I think we can put this, as we currently always have the folder.

File: egs2/nit_song070/svs1/run.sh (hunk 5 above)
Comment (ftshijt) [reply egs2/nit_song070/svs1/run.sh:10]: Just curious, is fmin used in following context besides pitch extraction?

Comment (A-Quarter-Mile) [reply egs2/nit_song070/svs1/run.sh:10]: The one used in pitch extraction is `f0min`. `fmin` is used in feats extraction.

File: .gitignore
Comment (wwwbxy123) [reply .gitignore:1]: midi_dump is not generated in this recipe. Removed as it's unused.

File: egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp:1]: Done

File: egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp (hunk 3 above)
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp]: Done.

File: egs2/nit_song070/svs1/conf/tuning/train_xiaoice_noDP.yaml
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/conf/tuning/train_xiaoice_noDP.yaml:1]: Done.

File: egs2/nit_song070/svs1/run.sh (hunk 5 above)
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/run.sh:11]: Done.

File: egs2/nit_song070/svs1/run.sh (hunk 6 above)
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/run.sh:40]: Done

File: egs2/TEMPLATE/asr1/pyscripts/utils/format_score_scp.py (hunk 2 above)
Comment (wwwbxy123) [reply egs2/TEMPLATE/asr1/pyscripts/utils/format_score_scp.py]: The change is also unnecessary. Reverted the change.

File: egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp (hunk 4 above)
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/conf/tuning/train_naive_rnn.yaml.tmp]: The file is removed.

File: egs2/nit_song070/svs1/run.sh (hunk 1 above)
Comment (wwwbxy123) [reply egs2/nit_song070/svs1/run.sh]: Done.

File: egs2/nit_song070/svs1/local/data.sh [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+stage=1
+stop_stage=100
+fs=24000
+
+log "$0 $*"
+
+. utils/parse_options.sh || exit 1;
+
+if [ -z "${NIT_SONG070}" ]; then
+    log "Fill the value of 'HIT_SONG070' of db.sh"
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/local/data.sh:26]:
```suggestion
    log "Fill the value of 'NIT_SONG070' of db.sh"
```

File: egs2/nit_song070/svs1/local/data.sh [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    # The nit data should be downloaded from http://hts.sp.nitech.ac.jp/archives/2.3/HTS-demo_NIT-SONG070-F001.tar.bz2
+    # Terms from http://hts.sp.nitech.ac.jp/
+fi
+
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+    log "stage 1: Data preparaion "
+
+    mkdir -p score_dump
+    mkdir -p wav_dump
+    # we convert the music score to midi format
```
Comment (A-Quarter-Mile) [inline egs2/nit_song070/svs1/local/data.sh:47]:
```suggestion
```

**Comment by ftshijt** [comment]
It looks ready to merge. Since the CI is recently blocking, we will wait a bit more time for it to be fixed. Thanks for your effort and contribution!

File: egs2/TEMPLATE/asr1/db.sh [hunk 9]
```diff
@@ -71,6 +71,7 @@ LJSPEECH=downloads
 MUSAN=
```
Comment (ftshijt) [inline egs2/TEMPLATE/asr1/db.sh:71]: A final note, could you also add an entry in egs2/README.md for nit_song?

Comment (wwwbxy123) [reply egs2/TEMPLATE/asr1/db.sh:71]: Done
